compute/zonal_instance_group_manager/main.tf

```
@@ -25,38 +25,27 @@ resource "google_compute_instance_template" "default" {
   machine_type   = "e2-medium"
   can_ip_forward = false
 
-  tags = ["tag1", "tag2"]
-
   disk {
     source_image = data.google_compute_image.default.id
```

Test it!

You don't have to define `data.google_compute_image`. You can use as following.

```suggestion
    source_image = "debian-cloud/debian-11"
```

Example: https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/compute_instance_from_template

---

Thank you for that suggestion. I have done that (and removed the google_compute_image resource).